src/systems/tracked-controls.js

```diff
@@ -38,11 +36,23 @@ module.exports.System = registerSystem('tracked-controls', {
     var gamepads;
     var i;
     var prevCount;
+    var thisCount;
 
     gamepads = navigator.getGamepads && navigator.getGamepads();
     if (!gamepads) { return; }
 
+    // Count first, and only actually update controller list if length changes.
     prevCount = controllers.length;
+    thisCount = 0;
```

Comment:
I think this check does not save much and is brittle. Controllers might change (one controller connects and another disconnects) and the number remains the same